pdns/dnsdist-lua-actions.cc

```diff
@@ -1728,8 +1740,17 @@ void setupLuaActions(LuaContext& luaCtx)
       return ret;
     });
 
-  luaCtx.writeFunction("SpoofRawAction", [](const std::string& raw, boost::optional<responseParams_t> vars) {
-      auto ret = std::shared_ptr<DNSAction>(new SpoofAction(raw));
+  luaCtx.writeFunction("SpoofRawAction", [](boost::variant<std::string,vector<pair<int, std::string>>> inp, boost::optional<responseParams_t> vars) {
+      vector<string> raws;
+      if(auto s = boost::get<std::string>(&inp))
```

Comment:
Just a nit but I'd rather always have braces, it prevents mistakes if someone later wants to add a second instruction in that `if` case.

Reply:
Added, and as a bonus for you I also updated the place where I copy/pasted it from ;)